python/torch_mlir/dialects/torch/importer/jit_ir/csrc/torch_to_mlir_utils.cpp

```diff
@@ -319,6 +319,10 @@ MlirAttribute torch_mlir::convertTensorToMlirElementsAttr(at::Tensor tensor,
   case ScalarType::QInt8:
     return mlirDenseElementsAttrInt8Get(
         shapedType, numElements, static_cast<const int8_t *>(tensorData));
+  case ScalarType::BFloat16:
+    return mlirDenseElementsAttrRawBufferGet(shapedType,
```

Comment:
I think we should add mlirDenseElementsAttrBFloat16Get upstream.

Reply:
Thanks, this patch now uses the `mlirDenseElementsAttrBFloat16Get()` function from upstream LLVM.